apps/web/lib/getBooking.tsx

```diff
@@ -1,23 +1,25 @@
 import { Prisma, PrismaClient } from "@prisma/client";
 
+const bookingSelect = {
```

Comment:
This is not being reused anywhere. And we're losing the type validations. Let's revert this.

https://handbook.cal.com/engineering/best-practices/prefer-inferred-types-over-explicit-ones

Reply:
https://github.com/calcom/cal.com/pull/6858/commits/9e6b4127b6cce232e5a737ff682c7372364f85e4